nixos/modules/services/network-filesystems/s3fs-fuse.nix

```diff
@@ -0,0 +1,100 @@
+# Inspired by: https://discourse.nixos.org/t/how-to-setup-s3fs-mount/6283/2
+
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.s3fs-fuse;
+
+  mountModule = types.submodule {
+    options = {
+      mountPoint = mkOption {
+        type = types.str;
```

Comment:
```suggestion
        type = types.path;
```

Reply:
I think I tried using path, and if you used a path that did not exist, the rebuild would fail.
Am I wrong or am I missing something?